plugins/generator-1.19.2/forge-1.19.2/procedures/_return.java.ftl

```diff
@@ -0,0 +1 @@
+return ${opt.removeParentheses(value)};
```

Comment:
mcitems is now ported, so you can port full _return with itemstack and blockstate handlers